webpack.config.js

```diff
@@ -21,7 +22,7 @@ function getLoadersConfig(env) {
           'sass?includePaths[]=' + (path.resolve(__dirname, "./sass/themes", process.env.CONFIG ? process.env.CONFIG : 'default'))
```

Comment:
We can replace this with:
{ test: /\.scss$/,loaders: ['style', 'css', 'postcss', 'sass']},

We don't need includePaths anymore, right?